rakelib/openhab.rake

```diff
@@ -115,15 +115,25 @@ namespace :openhab do
   desc 'Download Openhab and unzip it'
   task download: [OPENHAB_DIR] do |task|
     state(task.name) do
+      download_url = case @openhab_version
+                     when /.*\.M\d/
+                       'https://openhab.jfrog.io/openhab/libs-milestone-local/org/openhab/distro/openhab/'\
+                       "#{@openhab_version}/openhab-#{@openhab_version}.zip"
+                     when /.*-SNAPSHOT/
+                       'https://ci.openhab.org/job/openHAB3-Distribution/lastSuccessfulBuild/artifact/'\
+                       "distributions/openhab/target/openhab-#{@openhab_version}.zip"
+                     else
+                       'https://openhab.jfrog.io/openhab/libs-release/org/openhab/distro/openhab/'\
+                       "#{@openhab_version}/openhab-#{@openhab_version}.zip"
+                     end
       openhab_zip = "openhab-#{@openhab_version}.zip"
```

Comment:
```suggestion
      openhab_zip = "openhab-#{@openhab_version}.zip"
      download_url = case @openhab_version
                     when /.*\.M\d/
                       'https://openhab.jfrog.io/openhab/libs-milestone-local/org/openhab/distro/openhab/'\
                       "#{@openhab_version}/#{@openhab_zip}"
                     when /.*-SNAPSHOT/
                       'https://ci.openhab.org/job/openHAB3-Distribution/lastSuccessfulBuild/artifact/'\
                       "distributions/openhab/target/#{@openhab_zip}"
                     else
                       'https://openhab.jfrog.io/openhab/libs-release/org/openhab/distro/openhab/'\
                       "#{@openhab_version}/#{@openhab_zip}"
                     end
```

Reply:
Good catch, this would DRY it up a little.